### ISSUE TITLE
JupyterLab Icons are not displayed in the storybook

### ISSUE THREAD

**Issue by echarles** [datalayer/jupyterlab-react-widgets#1]
Debugging the storybook in the [ui-components](https://github.com/jupyterlab/jupyterlab/tree/master/packages/ui-components) resolves [svgHtml](https://github.com/jupyterlab/jupyterlab/blob/fbaf591b7ad68eee44bb3b96b5f3a2f4f9370c11/packages/ui-components/src/icon/iconregistry.tsx#L210)
as a HTML content from file and creates correctly a [svgElement](https://github.com/jupyterlab/jupyterlab/blob/fbaf591b7ad68eee44bb3b96b5f3a2f4f9370c11/packages/ui-components/src/icon/iconregistry.tsx#L221), e.g. for the `build` icon:

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
    <path class="jp-icon3" fill="#616161" d="M22.7 19l-9.1-9.1c.9-2.3.4-5-1.5-6.9-2-2-5-2.4-7.4-1.3L9 6 6 9 1.6 4.7C.4 7.1.9 10.1 2.9 12.1c1.9 1.9 4.6 2.4 6.9 1.5l9.1 9.1c.4.4 1 .4 1.4 0l2.3-2.3c.5-.4.5-1.1.1-1.4z"/>
</svg>
```

This [jupyterlab-react-widgets](https://github.com/datalayer/jupyterlab-react-widgets) repository implements a storybook that depends on the built/released [ui-components](https://github.com/jupyterlab/jupyterlab/tree/master/packages/ui-components) package.

While trying different [webpack configs for this storybook](https://github.com/datalayer/jupyterlab-react-widgets/blob/master/.storybook/webpack.config.js) to load SVG as no loader, raw-loader, url-loader, svg-url-loader..., the created [svgHtml](https://github.com/jupyterlab/jupyterlab/blob/fbaf591b7ad68eee44bb3b96b5f3a2f4f9370c11/packages/ui-components/src/icon/iconregistry.tsx#L210) is resolved to something like `static/media/build.e84fc627.svg`.

With no SVG loader defined in `webpackconfig`, the icon is correctly displayed invoking `http://localhost:9001/static/media/build.7d1efce3.svg`

<img width="494" alt="Screenshot 2019-12-31 at 11 10 54" src="https://user-images.githubusercontent.com/226720/71618189-e2f97000-2bbe-11ea-9432-3c75166d22b7.png">

However, because the storybook changes the Browser URL based on the requested story (e.g. `http://localhost:9001/?path=/story/iconreact--build-icon`) and because the returned svgHtml is relative (not absolute), the requested URL is something like `http://localhost:9001/?path=/story/iconreact--build-iconstatic/media/build.7d1efce3.svg` and fails to render the SVG.

For now, SVG icons are not usable with Storybook and prolly with other application that handle non-roolt browser URLs.

Bottom line: Or there is a way to configure webpackconfig to achieve this or something must be changed in [ui-components](https://github.com/jupyterlab/jupyterlab/tree/master/packages/ui-components)?
